tests/support/server.tcl

```diff
@@ -536,7 +536,7 @@ proc start_server {options {code undefined}} {
     }
 
     # write new configuration to temporary file
-    set config_file [tmpfile redis.conf]
+    set config_file [tmpfile valkey.conf]
```

Comment:
this name also has compat risk. is the plan to symlink?

Reply:
Yes, but it depends which version this PR is targeting. Our next major version???

Reply:
I think creating symlink for all the valkey named binaries and conf file should be good for compatibility purposes. Do you folks see any issue with that?

Reply:
I am onboard with symlinking in the first release and either direction works for me (redis->valkey or valkey to redis)

Reply:
We decided to go with valkey.conf and no symlink, since this file is not installed by `make install`.